<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogFormakeAccount</class>
 <widget class="QDialog" name="DialogFormakeAccount">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>339</width>
    <height>229</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalName">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>*Name :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LineEditName"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontaluser">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>*Username :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LineEditUser"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalpass">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>*Pass :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LineEditPass"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutrepass">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>*re-pass :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LineEditPass_2"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutpass">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Email :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LineEditEmail"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutBottom">
     <item>
      <widget class="QPushButton" name="makeaccPush">
       <property name="text">
        <string>Makeaccount</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_push">
       <property name="text">
        <string>cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
